ansible/slurm.yml

```diff
@@ -15,14 +15,6 @@
   tags:
     - openhpc
   tasks:
-    - name: Add CentOS 8.3 Vault repo for OpenHPC hwloc dependency
-      # NB: REMOVE THIS once OpenHPC works on CentOS 8.4
-      yum_repository:
```

Comment:
Consider using: `state: absent` to remove this file on hosts upgraded from a previous version.